contribute.html

```diff
@@ -56,11 +56,21 @@
       <article>
         <section>
           <h3>We'd love your help</h3>
-          <p>Zulip welcomes all forms of contributions! There's a ton of stuff to do. Here's just a small sampling of ways you can get started:</p>
+          <p>Zulip is licensed under the Apache license and is a
+          friendly open source community that welcomes all forms of
+          contributions! There's a ton of stuff to do -- in
+          development, testing, translation, documentation, and
+          more. Here's just a small sampling of ways you can get
+          started:</p>
           <ul>
-            <li>Write an integration for your favorite service</li>
-            <li>Write a docker file to make Zulip available to dockerized servers</li>
+            <li><a href="https://zulip.readthedocs.io/en/latest/integration-guide.html">Write
+            an integration for your favorite service</a></li>
+            <li>Improve
+            our <a href="https://github.com/zulip/zulip/blob/master/Dockerfile">Dockerfile</a>
+            to make Zulip available to dockerized servers</li>
             <li>Add a service monitoring dashboard</li>
+	    <li><a href="https://zulip.readthedocs.io/en/latest/translating.html">Translate</a>
+	    user-visible words into another language</li>
             <li>Add your favorite emoji</li>
```

Comment:
I'll slim this page way down and link to http://zulip.readthedocs.io/en/latest/readme-symlink.html#ways-to-contribute -- good idea -- but I still think it's a good idea to have this as a standalone zulip.org page, one that's got our branding. And I like that we have an opportunity here to display the zulip-security@googlegroups.com address **on** zulip.org, where it's easy to find for someone who wants/needs it.